src/app/Library/CrudPanel/Traits/Create.php

```diff
@@ -28,7 +30,9 @@ public function create($data)
 
         // omit the n-n relationships when updating the eloquent item
         $nn_relationships = Arr::pluck($this->getRelationFieldsWithPivot(), 'name');
-        $item = $this->model->create(Arr::except($data, $nn_relationships));
+        $repeatable_relationships = Arr::pluck($this->getRelationFieldsWithRepeatable(), 'name');
+        $excluded_relationships = array_merge($nn_relationships, $repeatable_relationships);
+        $item = $this->model->create(Arr::except($data, $excluded_relationships));
```

Comment:
This could probably be simplified by having a CrudPanel method that gets fields relationships that are saveable. See https://github.com/Laravel-Backpack/CRUD/pull/3200#discussion_r487415554